content/docs/fibers/error_handling.mdz

```diff
@@ -32,8 +32,14 @@ from the last call to @code`resume`.
  (print "value returned: " res))
 ```
 
-Janet also provides a simple macro @code`try` to make this a bit easier in the
-common case.
+## Two error-handling forms built on fibers
+
+### @code`try`
+
+Janet provides a simple macro @code`try` to make error-handling a bit
+easier in the common case. @code`try` performs the function of a
+traditional try/catch block; it will execute its body and, if any
+error is raised, pattern match against the error in its second clause.
```

Comment:
I do not think it is pattern matched. Can you please elaborate on the idea? When looking at https://github.com/janet-lang/janet/blob/master/src/boot/boot.janet#L232, I do not see any pattern matching.

Reply:
There is indeed no pattern matching - there is just two optional bindings - the error itself and the fiber that raised the error.